test_priorityq.py

```diff
@@ -0,0 +1,53 @@
+import pytest
+import priorityq
+
+
+def test_insert_peek(empty_pqueue):
+    """Test adding items to the list with correct priority and peeking."""
+    # tests that adding lower priority doesn't change first
+    empty_pqueue.insert(1, 1)
+    assert empty_pqueue.peek() == 1
+    empty_pqueue.insert(2, 2)
+    assert empty_pqueue.peek() == 1
+    empty_pqueue.insert(3, 3)
+    assert empty_pqueue.peek() == 1
+    # test add value with priority greater than number initial priority range
+    empty_pqueue.insert(4, 15)
+    assert empty_pqueue.peek() == 1
+    # tests that lower priorities were inserted correctly
+    assert empty_pqueue.pop() == 1
+    assert empty_pqueue.pop() == 2
+    assert empty_pqueue.pop() == 3
+    assert empty_pqueue.pop() == 4
+    # test peeking at empty queue raises error
+    with pytest.raises(AttributeError):
+        empty_pqueue.peek()
+    # tests that adding higher priority is returned first
+    empty_pqueue.insert(1, 3)
+    assert empty_pqueue.peek() == 1
+    empty_pqueue.insert(2, 2)
+    assert empty_pqueue.peek() == 2
+    empty_pqueue.insert(3, 1)
+    assert empty_pqueue.peek() == 3
```

Comment:
There are too many assertions in this test.  If the first one fails, you have no idea if the later ones would have succeeded.  Try to keep your tests short and focused.